<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWidget</class>
 <widget class="QWidget" name="SettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="homeFontSizeLabel">
       <property name="text">
        <string>Home Font Size:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="homeFontSizeSpinBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="logoLabel">
       <property name="text">
        <string>Logo:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="selectLogoBtn">
       <property name="text">
        <string>Select...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="homeTextLabel">
       <property name="text">
        <string>Home Text:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="homeTextLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="currencySymbolLabel">
       <property name="text">
        <string>Currency Symbol:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="currencySymbolLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="productFontSizeLabel">
       <property name="text">
        <string>Product Font Size:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="productFontSizeSpinBox"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveBtn</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWidget</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>selectLogoBtn</sender>
   <signal>clicked()</signal>
   <receiver>SettingsWidget</receiver>
   <slot>selectLogo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>selectLogo()</slot>
 </slots>
</ui>
